### PR TITLE
update: java version note

### PR DESCRIPTION
**Pull request by danil-pavlov**
* [KT-68317](https://youtrack.jetbrains.com/issue/KT-68317/)
* [KT-49691](https://youtrack.jetbrains.com/issue/KT-49691/)

### CODE REVIEW

File: docs/topics/native/native-command-line-compiler.md [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
+Choose a working directory and create there a file named `hello.kt`. Add the following code:
 
 ```kotlin
 fun main() {
-  println("Hello Kotlin/Native!")
+  println("Hello, Kotlin/Native!")
 }
 ```
 
+The application will print "Hello, Kotlin/Native" to the standard output.
```
Comment (zamulla) [inline docs/topics/native/native-command-line-compiler.md:30]: At first glance I thought that this is what's going to happen after I created the file. The previous version of the section started with this, and it seems a clearer sequence: here's what we want the code to do, and here's the code itself, based on that description.

File: docs/topics/native/native-command-line-compiler.md [hunk 2]
```diff
@@ -36,8 +40,8 @@ kotlinc-native hello.kt -o hello
 
 The value of `-o` option specifies the name of the output file, so this call should generate a `hello.kexe` (Linux and macOS)
 or `hello.exe` (Windows) binary file.
-For the full list of available compiler options, see the [compiler options reference](compiler-reference.md).
+For the full list of available options, see [Kotlin compiler options](compiler-reference.md).
```
Comment (zamulla) [inline docs/topics/native/native-command-line-compiler.md:44]: Reading this, I would expect the outcome here, as in "and now let's run the program we compiled." But that's your call as to how necessary that is.

**Review by zamulla — CHANGES_REQUESTED** [review]
Looks good, I noted one potentially confusing sequence.

File: docs/topics/native/native-command-line-compiler.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/native/native-command-line-compiler.md]: True, we're not running the program yet at this point

File: docs/topics/native/native-command-line-compiler.md (hunk 2 above)
Comment (danil-pavlov) [reply docs/topics/native/native-command-line-compiler.md:42]: Added a separation section below

**Review by zamulla — APPROVED** [review]
Noice!

**Pull request merged**